it/it-tests/src/test/java/it/projectSearch/SearchProjectsTest.java

```diff
@@ -20,17 +20,20 @@
 package it.projectSearch;
 
 import com.sonar.orchestrator.Orchestrator;
-import com.sonar.orchestrator.build.SonarScanner;
```

Comment:
The commit message is > 72 characters, which makes it inconvenient for me to read on github. I suggest:

> SONAR-9377 fix facet values of projects page, when using text query